### ISSUE TITLE
Could you please share the configs of HRNet cityscapes?

### ISSUE THREAD

**Issue by nigyiii** [PaddlePaddle/PaddleSeg#245]
Hi, i found it hard to reproduce the results on cityscapes using HRNet w48. I trained the model with train set and mIoU is only about 70% on validation set. Can you share the config file(.yaml) of HRNet cityscapes?

**Comment by nigyiii** [comment]
Below is my config.
这是我的配置
```
DATASET:
    DATA_DIR: "./dataset/cityscapes/"
    IMAGE_TYPE: "rgb"  # choice rgb or rgba
    NUM_CLASSES: 19
    TEST_FILE_LIST: "dataset/cityscapes/test.list"
    TRAIN_FILE_LIST: "dataset/cityscapes/train.list"
    VAL_FILE_LIST: "dataset/cityscapes/val.list"
    IGNORE_INDEX: 255

# 预训练模型配置
MODEL:
    MODEL_NAME: "hrnet"
    DEFAULT_NORM_TYPE: "bn"
    HRNET:
        STAGE2:
            NUM_CHANNELS: [48, 96]
        STAGE3:
            NUM_CHANNELS: [48, 96, 192]
        STAGE4:
            NUM_CHANNELS: [48, 96, 192, 384]

# 其他配置
TRAIN_CROP_SIZE: (1024, 512)
EVAL_CROP_SIZE: (1024, 512)
AUG:
    FIX_RESIZE_SIZE: (1024, 512)
    MIRROR: True
BATCH_SIZE: 12
TRAIN:
    PRETRAINED_MODEL_DIR: "./pretrained_model/HRNet_W48_C_pretrained/"
    MODEL_SAVE_DIR: "./saved_model/hrnet_cityscapes_w48_train/"
    SNAPSHOT_EPOCH: 5
    # 是否使用多卡间同步BatchNorm均值和方差
    SYNC_BATCH_NORM: True
TEST:
    TEST_MODEL: "./saved_model/hrnet_cityscapes_w48_train/final"
SOLVER:
    NUM_EPOCHS: 100
    LR: 0.01
    LR_POLICY: "poly"
    OPTIMIZER: "sgd"
    # 动量参数
    MOMENTUM: 0.9
    # 学习率权重衰减，0-1
    WEIGHT_DECAY: 0.0005
```

**Comment by wuyefeilin** [comment]
@nigyiii 
按照下面试一下：
```
DATASET:
    DATA_DIR: "./dataset/cityscapes/"
    IMAGE_TYPE: "rgb"  # choice rgb or rgba
    NUM_CLASSES: 19
    TEST_FILE_LIST: "dataset/cityscapes/test.list"
    TRAIN_FILE_LIST: "dataset/cityscapes/train.list"
    VAL_FILE_LIST: "dataset/cityscapes/val.list"
    IGNORE_INDEX: 255

# 预训练模型配置
MODEL:
    MODEL_NAME: "hrnet"
    DEFAULT_NORM_TYPE: "bn"
    HRNET:
        STAGE2:
            NUM_CHANNELS: [48, 96]
        STAGE3:
            NUM_CHANNELS: [48, 96, 192]
        STAGE4:
            NUM_CHANNELS: [48, 96, 192, 384]

# 其他配置
TRAIN_CROP_SIZE: (1024, 512)
EVAL_CROP_SIZE: (2048, 1024)
AUG:
    AUG_METHOD: 'stepscaling
    MIRROR: True
BATCH_SIZE: 12
TRAIN:
    PRETRAINED_MODEL_DIR: "./pretrained_model/HRNet_W48_C_pretrained/"
    MODEL_SAVE_DIR: "./saved_model/hrnet_cityscapes_w48_train/"
    SNAPSHOT_EPOCH: 5
    # 是否使用多卡间同步BatchNorm均值和方差
    SYNC_BATCH_NORM: True
TEST:
    TEST_MODEL: "./saved_model/hrnet_cityscapes_w48_train/final"
SOLVER:
    NUM_EPOCHS: 500
    LR: 0.01
    LR_POLICY: "poly"
    OPTIMIZER: "sgd"
    # 动量参数
    MOMENTUM: 0.9
    # 学习率权重衰减，0-1
    WEIGHT_DECAY: 0.0005
```

**Comment by InternetMaster1** [comment]
@wuyefeilin  @ZeyuChen  @nigyiii 

Is this HRNetV2-W18-SMALL-v2 from the [official library](https://github.com/HRNet/HRNet-Semantic-Segmentation)?

Official Library has mIoU of 76.2 on CityScapes for HRNetV2-W18-SMALL-v2

model | Train Set | Test Set | #Params | GFLOPs | OHEM | Multi-scale | Flip | Distillation | mIoU | Link
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --
HRNetV2-W18-Small-v1 | Train | Val | 1.5M | 31.1 | No | No | No | No | 70.3 | OneDrive/BaiduYun(Access Code:63be)
HRNetV2-W18-Small-v2 | Train | Val | 3.9M | 71.6 | No | No | No | No | 76.2 | OneDrive/BaiduYun(Access Code:p1qf)


Your Library [Model Zoo](https://github.com/PaddlePaddle/PaddleSeg/blob/develop/docs/model_zoo.md) has mIoU of 79.36 on CityScapes for HRNet_W18 / bn

model | Data collection | download link | Output Stride | mutli-scale test | mIoU on val
-- | -- | -- | -- | -- | --
HRNet_W18 / bn | Cityscapes | hrnet_w18_bn_cityscapes.tgz | 4 | false | 0.7936

What are the number of parameters in "HRNet_W18 / bn"?

I am looking to implement lightweight semantic segmentation on a mobile device for highly accurate human cutouts (full body, not just portrait). What would you recommend?

**Comment by wuyefeilin** [comment]
We have answered in https://github.com/PaddlePaddle/PaddleSeg/issues/259.

**Comment by InternetMaster1** [comment]
Thanks
